tools/ports/contrib/glfw3.py

```diff
@@ -6,8 +6,8 @@
 import os
 from typing import Dict
 
-TAG = '1.0.5'
-HASH = '4ef49ad23bd70a03639df4c8b0fdfe5b8c25bfe4d183c52a7e2d6e7285ab23c09b688b898911f0b9ce0fd014840a73c220592c01bf6e3af5280d927a2718f8aa'
+TAG = '1.1.0'
```

Comment:
I wonder if it would make sense to use a versioning scheme that reflected that upstream version.  e.g. `3.4.0-v1`?

Reply:
I will think about it for future updates.